shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -856,3 +856,31 @@ async fn test_protocol_v5_single(#[case] driver: CassandraDriver) {
 
     shotover.shutdown_and_then_consume_events(&[]).await;
 }
+
+#[rstest]
+#[case::cdrs(CdrsTokio)]
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_protocol_v5_compression(#[case] driver: CassandraDriver) {
+    {
+        let _docker_compose =
+            docker_compose("example-configs/cassandra-passthrough/docker-compose.yaml");
+
+        let shotover = ShotoverProcessBuilder::new_with_topology(
+            "example-configs/cassandra-passthrough/topology-encode.yaml",
```

Comment:
We also need to test `cassandra-passthrough/topology.yaml`, in order to properly test the encoder.